src/pages/Sketch/components/Canvas.jsx

```diff
@@ -493,27 +498,45 @@ function Canvas() {
       });
   };
 
-  const initiateLoadSaved = () => document.getElementById("file-selector").click();
-  const loadLastState = e => {
+
+  
+ 
+  const initiateload = () => document.getElementById("file-selector").click();
+  const loadimport= e =>{   
```

Comment:
**You renamed the d2ps loading function to make your own image loading funciton**